lib/ruby_zen/indexers/iseq_indexer.rb

```diff
@@ -40,7 +40,7 @@ def register_processors
         const = @engine.fetch_class(name)
         if const.nil?
           @engine.define_class(name) do
-            RubyZen::ClassObject.new(name, is_module: true)
+            RubyZen::ClassObject.new(name)
```

Comment:
Thanks to the class definition update, we can now be sure about the class type (module or normal class). So, I think we still need to assume that an unknown class is a module, until we have better evidence, like what Ruby is doing.